doc/source/guides/common-tasks.rst

```diff
@@ -29,11 +30,12 @@ document from an image.
 
     ...
 
-    // Assuming that `gini` is an instance of the `GiniSDK` facade class and `image` is a `UIImage` instance,
-    // e.g. from a picture taken by the camera.
+    // Assuming that `gini` is an instance of the `GiniSDK` facade class, 
+    // `docType` is a string containing a hint about the type of document and `image` is a `NSData` instance,
```

Comment:
an `NSData` instance ('an' rather than 'a'; because of the phonetics of the following word which is pronounced EN-ES-Data :-) )